src/subcommand/server.rs

```diff
@@ -1660,85 +1637,60 @@ impl Server {
         }
       }
 
-      let info = Index::inscription_info(&index, query)?
+      let (info, txout, inscription) = index
+        .inscription_info(query)?
         .ok_or_not_found(|| format!("inscription {query}"))?;
 
-      let effective_mime_type = if let Some(delegate_id) = info.inscription.delegate() {
-        let delegate_result = index.get_inscription_by_id(delegate_id);
-        if let Ok(Some(delegate)) = delegate_result {
-          delegate.content_type().map(str::to_string)
-        } else {
-          info.inscription.content_type().map(str::to_string)
-        }
-      } else {
-        info.inscription.content_type().map(str::to_string)
-      };
-
       Ok(if accept_json {
-        Json(api::Inscription {
-          address: info
```

Comment:
@danadi7

Reply:
Does this work without the custom Ordzaar changes?